server/storage/sql_models.go

```diff
@@ -32,9 +32,9 @@ func (g TUFFile) TableName() string {
 	return TUFFileTableName
 }
 
-// Change defines the the fields required for an object in the changefeed
-type Change struct {
-	ID        uint `gorm:"primary_key" sql:"not null" json:",string"`
+// SQLChange defines the the fields required for an object in the changefeed
+type SQLChange struct {
+	ID        uint `gorm:"primary_key" sql:";not null" json:",string"`
```

Comment:
is the added `;` intended?

Reply:
good catch. I initially tried some fiddling here and must have left this in by accident. It will still work as this is a valid tag format for sql but I'll remove it for consistency.